src/content/React-CSharp/ClientApp/package.json

```diff
@@ -67,5 +67,13 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "resolutions": {
+    "ejs": ">=3.1.6",
+    "immer": ">=9.0.6",
+    "url-parse": ">=1.5.0",
+    "lodash": ">=4.17.21",
+    "glob-parent": ">=5.1.2",
+    "set-value": "4.0.1"
```

Comment:
Our templates use NPM, these don't do anything

Reply:
Updated

Reply:
I apologize will, I'm in the middle of doing a dep update (the traditional way) myself, just want to make sure I don't waste your time.